Dockerfile

```diff
@@ -25,9 +29,9 @@ ENV USER_UID=1000
 ENV USER_NAME=srt
 ENV EXECUTABLE_NAME=srt-game-server.bin
 ENV EXECUTABLE=/home/${USER_NAME}/bin/${EXECUTABLE_NAME}
-ENV BROKER_URI="tcp://127.0.0.1:5672"
-ENV LOG_LEVEL="1"
-ENV SLEEP_CYCLE="1500"
+ENV BROKER_URI=tcp://artemiscloud:5672
```

Comment:
@cybtachyon Not sure if this change makes sense in the default case for folks not using docker compose. There are likely others like myself who predominantly use podman and are generally not able to readily use a docker compose file.

Is there a way that this can be overridden or injected when the compose is in play?

Reply:
`podman` can be used with compose -- example here:

https://www.redhat.com/sysadmin/podman-docker-compose

I wouldn't be opposed to using it, but it would likely mean all containers end up having to be run as root (not sure with Fedora 34).

Reply:
When I get a chance to update I'll let you know :)

> [rkieley@i5t450s ~]$ podman --version
> podman version 1.8.0
> [rkieley@i5t450s ~]$ cat /etc/redhat-release 
> Fedora release 30 (Thirty)

> [rkieley@i7t450s ~]$ podman --version
> podman version 2.0.4
> [rkieley@i7t450s ~]$ cat /etc/redhat-release 
> Fedora release 31 (Thirty One)
> [rkieley@i7t450s ~]$

At the moment I'm not set up to do rootless so that wouldn't be a blocker, although I think maintaining compatibility with stuff that's some amount from leading edge would potentially be good too - maybe N - 1? So F33 timewise-ish?

Reply:
@RoddieKieley Have you tested this? I'm not using podman, but podman should be creating `/etc/hosts` so that `artmeiscloud` resolves in the dns.

Might need to update the command to add `hostname` though, can you let me know and we can add the README change to the PR:

```
sudo podman run --name artemis --hostname artemiscloud -it --rm -p 5672:5672 \
-e AMQ_USER=admin -e AMQ_PASSWORD=admin -e AMQ_ALLOW_ANONYMOUS=true \
quay.io/artemiscloud/activemq-artemis-broker:latest
```